experiments/extract_all_data.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
```

Comment:
Add a note that THIS FILE MUST BE PLACED IN THE bin/ DIRECTORY OF THE ARTIFACT BEFORE RUNNING or something along those lines

Reply:
maybe even copy the file in automatically? idk

Reply:
Added a small note in [8d01421](https://github.com/mlb2251/stitch/pull/87/commits/8d014211315a147353c291495e133bbb1dfdd7ad). Copying is a good idea since this is currently a bit error prone (e.g. one might change data_extractor.py in experiments/ and then forget to copy the new version over. However, that would either mean making an assumption on the relative location of the "correct" data_extractor.py or using something like `find` to search for it. This seems very over-engineered to me. After all the script is just an example of how to wrap around `data_extractor.py`, so I don't want to make it more complex than it needs to be